package.json

```diff
@@ -12,7 +12,7 @@
   "dependencies": {
```

Comment:
Version bump needed on line 3. As I think this is a release that will break previous versions, this release would bump to `2.0.0`. See https://semver.org/#summary for more info.

Reply:
So we have another PR where the version goes to 1.1.0, and this to 2.0.0, and then we have that mongo bump. What makes sense to version for each of these?

Reply:
After discussion, decided to go to 2.0.0 for both ES and MongoDB updates (breaking changes, but can go under the same version).